catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -165,14 +172,38 @@ public void setResultShow(List<String> resultShow) {
 
     public void setAttributeAliases(List<String> attributeAliases) {
         this.attributeAliases = attributeAliases.stream()
-                .map(str -> str.split("="))
+                .map(str -> str.split("=", 2))
```

Comment:
This would allow "`=`" signs inside the right side of the alias without error. Could that potentially corrupt aliases?

Reply:
What do you mean by "corrupt aliases"?

Reply:
Yep the intention was to just go ahead and allow an equals sign in the alias' value, which shouldn't mess anything up all the way through to the ui.

Reply:
@mackncheesiest, I wasn't sure how the aliases were used and if they were expected in any later code to not contain an "`=`". It just seemed like something that might be a potential problem, though I was guessing that without being sure about the context. According to the previous comment, it's not a problem.